lib/ddtrace/contrib/rails/framework.rb

```diff
@@ -61,11 +62,8 @@ def self.config_with_defaults(datadog_config)
           # We set defaults here instead of in the patcher because we need to wait
           # for the Rails application to be fully initialized.
           datadog_config[:rails].tap do |config|
-            config[:service_name] ||= (Datadog.configure.service_without_fallback || Utils.app_name)
-            config[:database_service] ||= "#{config[:service_name]}-#{Contrib::ActiveRecord::Utils.adapter_name}"
-            config[:controller_service] ||= config[:service_name]
-            config[:cache_service] ||= "#{config[:service_name]}-cache"
-            config[:job_service] ||= "#{config[:service_name]}-#{Contrib::ActiveJob::Ext::SERVICE_NAME}"
+            config[:database_service] ||= "#{datadog_config.service}-#{Contrib::ActiveRecord::Utils.adapter_name}"
+            config[:cache_service] ||= "#{datadog_config.service}-cache"
```

Comment:
I'm curious -- why keep the suffixed name for the `database_service` and the `cache_service`, but not the `job_service`?

Reply:
This whole part when poof